lib/src/cli/cli.dart

```diff
@@ -176,9 +176,7 @@ extension _Set on Set<String> {
     if (segments.intersection(this).isNotEmpty) return true;
 
     for (final value in this) {
-      if (Glob(value).matches(entity.path)) {
-        return true;
-      }
+      return value.isNotEmpty && Glob(value).matches(entity.path);
     }
```

Comment:
This wont ever loop over multiple values, as it will return on the first loop. That is why we had the `if` statement there.

Reply:
Indeed, thanks for noticing it! Completely forgot space-separated globs are accepted.